rp2040-hal/src/i2c.rs

```diff
@@ -23,6 +23,14 @@ use rp2040_pac::{I2C0, I2C1, RESETS};
 pub enum Error {
     /// I2C abort with error
     Abort(u32),
+    /// User passed in a read buffer that was 0 or >255 length
+    InvalidReadBufferLength(u32),
```

Comment:
u32 should be usize here, as that's the type for a slice length.

Reply:
The buffer length enum variants are now usize too.